genai/text_generation/textgen_with_multi_local_img.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package text_generation shows examples of generating text using the GenAI SDK.
+package text_generation
+
+// [START googlegenaisdk_textgen_with_multi_local_img]
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	genai "google.golang.org/genai"
+)
+
+// generateWithMultiLocalImg shows how to generate text using multiple local images as inputs.
+func generateWithMultiLocalImg(w io.Writer) error {
+	ctx := context.Background()
+
+	client, err := genai.NewClient(ctx, &genai.ClientConfig{
+		HTTPOptions: genai.HTTPOptions{APIVersion: "v1"},
+	})
+	if err != nil {
+		return fmt.Errorf("failed to create genai client: %w", err)
+	}
+
+	// TODO(Developer): Update with paths to your image files.
+	imageBytes1, err := os.ReadFile("./testdata/latte.jpg")
```

Comment:
issue(blocking): pass local file paths as arguments to sample. Don't embed test file paths into the sample itself.

Reply:
Here and below.